dot-studio/networking

```diff
@@ -41,19 +41,20 @@ document "configure_host" <<DOC
 DOC
 function configure_host() {
   # Return if the hostname to configure ($1) is an ipaddress
-  [[ "$1" =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]] && return 1;
+  [[ "$1" =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]] && return 1
```

Comment:
This strikes me as gnarly overkill for a test that will still return true for things that aren't legit IP addresses such as `999.999.999.999` (not to mention IPv6). I'd suggest the simpler approach (and getting rid of the `[[ … ]] && return 1` paradigm):
```bash
  if [[ -z $(tr -d "[:digit:].:" <<< "$1") ]]; then
    return 1
  fi
```